drivers/bridge/bridge.go

```diff
@@ -380,13 +380,6 @@ func (d *driver) configure(option map[string]interface{}) error {
 		return &ErrInvalidDriverConfig{}
 	}
 
-	if config.EnableIPForwarding {
```

Comment:
`config.EnableIPForwarding` is set based on the daemon flag and not by checking the sysctl `ip_forward`. By doing the way you are suggesting we will call `SetDefaultPolicy` even if `ip_forward` is already enabled; ie: `ip_forward` might be enabled and the default policy set be `ACCEPT`. Docker changing that to `DROP` is incorrect.